addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -220,6 +255,76 @@ function ComposerFactory({ Entity }) {
             return [...new Set(allPartnerIds)];
         }
 
+        _getFirstMentionSuggestion() {
+            this.active_id = this.mentionSuggestions[0].id;
+        }
+
+        _getLastMentionSuggestion() {
+            this.active_id = this.mentionSuggestions[this.mentionSuggestions.length - 1].id;
+        }
```

Comment:
I thought they were unused, but in fact used by composer component. So they should be public.
Also these methods are dangerous when there are no mention suggestions on this composer!
(and this is weird for a getter to have side-effect, here auto-assign `active_id`, which BTW also by-passes the `update` flow of an entity... :') )